### DISCUSSION TITLE
Build multiple exes using the same _internal, is it possible?

### DISCUSSION THREAD

**Discussion by jr23mx** [pyinstaller/pyinstaller#8918] [Help]
Hi, 
I would like create multiple exes in the same folder, using all of them the same folder _internal

If that is possible, how to make the spec file?

currently I am doing them one by one using something like this

# -*- mode: python ; coding: utf-8 -*-

block_cipher = None

# -*- Main - Inspekten
#
a = Analysis(['main.py'],
             pathex=['C:\\......'],
             binaries=[],
             datas=[
			('C:\\......','.'),			 
			('C:\\......','.'),
			('C:\\......','.')
			
			],
             hiddenimports=[],
             hookspath=[],
             runtime_hooks=[],
             excludes=['PyQt5', 'PySide2'],
             win_no_prefer_redirects=False,
             win_private_assemblies=False,
             cipher=block_cipher,
             noarchive=False)
pyz = PYZ(a.pure, a.zipped_data,
             cipher=block_cipher)
			 
exe = EXE(pyz,
          a.scripts,
          [],
          exclude_binaries=True,
          name='Inspekten',
          debug=False,
          bootloader_ignore_signals=False,
          strip=False,
          upx=True,
          console=False,
          icon='C:\\......',
		  version='C:\\......\\file_version_info.txt')
	  
		  
coll = COLLECT(exe,

               a.binaries,
               a.zipfiles,
               a.datas,
			   
               strip=False,
               upx=True,
               upx_exclude=[],
               name='bin')

then I use on the prompt:

pyarmor pack -s "C:\Users\........\Main.spec" main.py

I have tried many modes but I cannot make multiple py files into a exe, in the same folder.

**Comment by bwoodsend** [comment]
https://pyinstaller.org/en/stable/spec-files.html#multipackaging-with-one-folder-apps
